src/Assets/Javascript/buildHTML.js

```diff
@@ -0,0 +1,228 @@
+import { postLike, postComment } from './post';
+import { storeInfo, retrieveInfo } from './localStorage';
+import { getComments } from './get';
+import { countItems, countComments } from './itemsCounter';
+
+const involvementApi = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/LnQtP7rZrNpR2zDEqCBJ/comments';
+
+function htmlBuilder(obj) {
+  for (let i = 0; i < obj.length; i += 1) {
+    if (obj[i].length === 2) {
+      obj[i][0].appendChild(obj[i][1]);
+    } else if (obj[i].length === 3) {
+      const [a, b, c] = obj[i];
+      a.appendChild(b);
+      b.className = c;
```

Comment:
- [ ] Please avoid using `function` keyword instead use callback functions and promises in your codes